### PR TITLE
Import recipe

### PR DESCRIPTION
**Pull request by arnekuchenbecker**

### CODE REVIEW

File: app/src/main/java/com/scouts/kitchenplaner/model/usecases/ImportRecipe.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        onFailure: (Int, String) -> Unit,
+        onSuccess: (Recipe) -> Unit
+    ) {
+        val response = chefkochAPIService.getRecipe(id)
+        if (response.isSuccessful) {
+            val chefkochRecipe = response.body()
+            if (chefkochRecipe != null) {
+                val uri = Uri.parse(chefkochRecipe.previewImageUrlTemplate.replace(FORMAT_STRING, FORMAT_INPUT))
+                val recipeName = if (chefkochRecipe.subtitle.isNotBlank()) {
+                    "${chefkochRecipe.title} - ${chefkochRecipe.subtitle}"
```
Comment (kittyahy) [inline app/src/main/java/com/scouts/kitchenplaner/model/usecases/ImportRecipe.kt:43]: Let the subtitle be the description

**Review by kittyahy — COMMENTED** [review]
We should think about, that it should be possible to change the ingredients after adding them in the recipe screen, s.t. one can adjust the measurements when recipes get imported

**Comment by arnekuchenbecker** [comment]
> We should think about, that it should be possible to change the ingredients after adding them in the recipe screen, s.t. one can adjust the measurements when recipes get imported

Created issue #107 for this

**Comment by arnekuchenbecker** [comment]
I realised that the usecase class really shouldn't depend on the retrofit class (in particular it shouldn't require knowledge about the DTO and HTTP responses and so on). Created issue #108 to change this by extracting to a repository class.

**Review by kittyahy — APPROVED** [review]

**Pull request merged**